runtests.sh

```diff
@@ -0,0 +1 @@
+go test -v $(go list ./... | grep -v /vendor/)
```

Comment:
This is no longer needed in 1.9 though right?

https://golang.org/doc/go1.9#vendor-dotdotdot

Reply:
yes! Finally ("By popular request..." 😄  ). However, I don't think 1.9 is mainstream yet. It's only been out for 2 months.

Reply:
Bad link. I meant this one https://github.com/openfaas/faas-cli/blob/master/Makefile

Reply:
done